src/CMakeLists.txt

```diff
@@ -14,4 +14,14 @@ idf_component_register(SRCS
                             lwip
                             mdns
                             nvs_flash
+                            spiffs          # esp_spiffs
                         )
+
+set(WEB_SRC_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../frontend")
+
+if (EXISTS ${WEB_SRC_DIR}/dist)
```

Comment:
I wonder if there's a way to check if the website build files have changed since the last flash so that we don't keep writing the same data over and over again. Could be its own issue, but I doubt we'll have time to figure it out.

Reply:
Yes, I think this should be explored, but in its own issue.